sonar-batch/src/main/java/org/sonar/batch/issue/DeprecatedIssueAdapterForFilter.java

```diff
@@ -151,8 +156,14 @@ public boolean isNew() {
     throw unsupported();
   }
 
+  @Deprecated
   @Override
   public Duration debt() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)